ocpairprogramming/src/main/java/com/pino/project/ocpairprogramming/java8/ocp/chapter3/generics/UpdatingConditionally.java

```diff
@@ -0,0 +1,48 @@
+package com.pino.project.ocpairprogramming.java8.ocp.chapter3.generics;
```

Comment:
here we should check what happens when value is null, mapper is null and value in map is null.
This case is not considered in the book.

Reply:
Again, that's the main reason I've created this repository for. Should everyone feel free to share his thoughts.